aiven/client/cli.py

```diff
@@ -6057,6 +6060,54 @@ def byoc__cloud__permissions__remove(self) -> None:
             )
         )
 
+    @staticmethod
+    def add_prefix_byoc_resource_tag(tags: Mapping[str, S]) -> Mapping[str, S]:
```

Comment:
nit: At first glance, the naming of `add_*` and `remove_*` gave me the impression that these methods are mutating the  tags. But they are returning a new dict instead.
maybe a better naming would be `get_prefixed_byoc_resource_tag` or `with_prefixed_byoc_resource_tag()`

Reply:
I will rename them `add_prefix_to_keys()` and `remove_prefix_from_keys()`